### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by mativanarquero**
- Remove unused LazyHeader
- Fix line spacing
cc

### CODE REVIEW

**Review by swengr-janan — APPROVED** [review]
Clean!

File: src/pages/Home/HeaderSection/index.tsx [hunk 1]
```diff
@@ -39,9 +35,8 @@ const Section: React.FC = () => {
...
             </ActionsDiv>
         </StyledContainer>
-        </HeroHeader>
-        
+        </HeroHeader>        
     );
 };
 
-export default Section;
+export default Section;
```
Comment (kbaldo) [inline src/pages/Home/HeaderSection/index.tsx:42]: Kindly put a newline character after this line

**Review by kbaldo — COMMENTED** [review]
I hope it's not yet too late

File: src/pages/Home/HeaderSection/index.tsx (hunk 1 above)
Comment (swengr-janan) [reply src/pages/Home/HeaderSection/index.tsx:42]: Ayt! Thanks @kbaldo , revert ko nalang

Comment (kbaldo) [reply src/pages/Home/HeaderSection/index.tsx:42]: oki @swengr-janan

Comment (mativanarquero) [reply src/pages/Home/HeaderSection/index.tsx:42]: Done. Thank you!

Comment (swengr-janan) [reply src/pages/Home/HeaderSection/index.tsx:42]: masasabay ba yun sa merging? not familiar pa sa process ni pag revert.

Comment (kbaldo) [reply src/pages/Home/HeaderSection/index.tsx:42]: Hmm Im not sure. New PR nalang, I think. Haha

**Pull request merged**